### PR TITLE
Try to learn git, orginize to "kingly_keys" subfolder, and add support for various keyboards.

### PR DESCRIPTION
**Pull request by TheRoyalSweatshirt**
…oard support

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by TheRoyalSweatshirt** [comment]
I couldn't/can't figure out exactly how @drashna wanted me to format the OLED code for the RoMac_Plus firmware.

**Comment by drashna** [comment]
Looks like romac and ropro boards didn't get moved? 

You may need to run
```
git rm -r keyboards/ropro keyboards/romac
```
This way, git removes the old folders, and moves them to the now location.

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	[FN] = LAYOUT(
+		KC_TRNS, KC_HOME, KC_PGUP, \
+		KC_TRNS, KC_END, KC_PGDN, \
+		KC_TRNS, KC_TRNS, KC_TRNS, \
+		KC_TRNS, KC_TRNS, KC_ENT \
+	)
+};
+
+#ifdef OLED_DRIVER_ENABLE
+void oled_task_user(void) {
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
oled_rotation_t oled_init_user(oled_rotation_t rotation) {
    return OLED_ROTATION_270;  // flips the display 180 degrees if offhand
}

void oled_task_user(void) {
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  // Host Keyboard Layer Status
+  oled_write_P(PSTR("Let's build something, together!"), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(PSTR("Let's\nbuild\nsome-\nthing\nto-\nget-\nher!"), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  switch (get_highest_layer(layer_state)) {
+    case BASE:
+      oled_write_P(PSTR("\n"), false);
+      break;
+    case FN:
+      oled_write_P(PSTR("FN\n"), false);
+      break;
+    default:
+      // Or use the write_ln shortcut over adding '\n' to the end of your string
+      oled_write_ln_P(PSTR("Undefined"), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
      oled_write_ln_P(PSTR("Undef"), false);
```

File: keyboards/kingly_keys/romac_plus/romac_plus.c [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#include "romac_plus.h"
+
+#ifdef __attribute__((weak)) OLED_DRIVER_ENABLE
```
Comment (TheRoyalSweatshirt) [inline keyboards/kingly_keys/romac_plus/romac_plus.c]: How should this be formatted?

Comment (drashna) [reply keyboards/kingly_keys/romac_plus/romac_plus.c]: This shouldn't be needed at all, actually.

**Comment by TheRoyalSweatshirt** [comment]
Oh my god it’s beautiful

File: keyboards/kingly_keys/little_foot/rules.mk [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
```
Comment (fauxpark) [inline keyboards/kingly_keys/little_foot/rules.mk]: These lines can be deleted.

File: keyboards/kingly_keys/romac/rules.mk [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac/rules.mk]: If you know the bootloader for this keyboard, please replace this block with:

```make
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define FN 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[BASE] = LAYOUT(
+		KC_7, KC_8, KC_9, \
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+      oled_write_P(PSTR("FN\n"), false);
+      break;
+    default:
+      // Or use the write_ln shortcut over adding '\n' to the end of your string
+      oled_write_ln_P(PSTR("Undef"), false);
+  }
+
+  // Host Keyboard LED Status
+  uint8_t led_usb_state = host_keyboard_leds();
+  oled_write_P(led_usb_state & (1<<USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+  oled_write_P(led_usb_state & (1<<USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  oled_write_P(led_usb_state & (1<<USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```

File: keyboards/kingly_keys/soap/rules.mk [hunk 11]
```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/kingly_keys/soap/rules.mk]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/kingly_keys/smd_milk/readme.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+# 2% Milk
+
+![SMD_MILK](https://cdn.discordapp.com/attachments/527180441950617615/628391976001208360/image0.jpg)
```
Comment (fauxpark) [inline keyboards/kingly_keys/smd_milk/readme.md]: Might be better to put this on imgur instead, I don't know how long Discord keeps images around for.

File: keyboards/kingly_keys/ropro/rules.mk [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u4
+
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/kingly_keys/ropro/rules.mk]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/kingly_keys/romac_plus/rules.mk [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (fauxpark) [inline keyboards/kingly_keys/romac_plus/rules.mk]: These lines can be deleted.

File: keyboards/kingly_keys/little_foot/little_foot.h [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+          k42,        k45,      k47,       k48        \
+) \
+{ \
+    { k01, k02, k03, k04, k05, k06, k07, k08, k09, k010 }, \
+    { k11, k12, k13, k14, k15, k16, k17, k18, k19, k110 }, \
+    { k21, k22, k23, k24, k25, k26, k27, k28, k29, k210 }, \
+    { k31, k32, k33, k34, k35, k36, k37, k38, k39, k310 }, \
+    { XXX, k42, XXX, XXX, k45, XXX, k47, k48, XXX,  XXX } \
+}
+
```
Comment (noroadsleft) [inline keyboards/kingly_keys/little_foot/little_foot.h]: `LAYOUT_big_space_base` needs a macro here.

File: keyboards/kingly_keys/little_foot/info.json [hunk 16]
```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard_name": "Little Foot",
+  "url": "",
+  "maintainer": "TheRoyalSweatshirt",
+  "width": 10,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_split_space_base": {
+      "key_count": 44,
+      "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":4, "y":2}, {"x":5, "y":2}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}, {"x":4, "y":3}, {"x":5, "y":3}, {"x":6, "y":3}, {"x":7, "y":3}, {"x":8, "y":3},, {"x":9, "y":3}, {"x":1.5, "y":4, "w":1.5}, {"x":3, "y":4, "w":2}, {"x":5, "y":4, "w":2}, {"x":7, "y":4, "w":1.5}]
```
Comment (noroadsleft) [inline keyboards/kingly_keys/little_foot/info.json]: Fixes invalid JSON (an extra comma).

```suggestion
      "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":4, "y":2}, {"x":5, "y":2}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}, {"x":4, "y":3}, {"x":5, "y":3}, {"x":6, "y":3}, {"x":7, "y":3}, {"x":8, "y":3}, {"x":9, "y":3}, {"x":1.5, "y":4, "w":1.5}, {"x":3, "y":4, "w":2}, {"x":5, "y":4, "w":2}, {"x":7, "y":4, "w":1.5}]
```

File: keyboards/kingly_keys/little_foot/info.json [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+    },
+      "LAYOUT_big_space_base": {
+        "key_count": 41,
+        "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":4, "y":2}, {"x":5, "y":2}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}, {"x":4, "y":3}, {"x":5, "y":3}, {"x":6, "y":3}, {"x":7, "y":3}, {"x":8, "y":3},, {"x":9, "y":3}, {"x":1.5, "y":4, "w":7}]
```
Comment (noroadsleft) [inline keyboards/kingly_keys/little_foot/info.json]: Fixes invalid JSON (an extra comma and a missing curly bracket).

```suggestion
        "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6, "y":0}, {"x":7, "y":0}, {"x":8, "y":0}, {"x":9, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1}, {"x":5, "y":1}, {"x":6, "y":1}, {"x":7, "y":1}, {"x":8, "y":1}, {"x":9, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":4, "y":2}, {"x":5, "y":2}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}, {"x":4, "y":3}, {"x":5, "y":3}, {"x":6, "y":3}, {"x":7, "y":3}, {"x":8, "y":3}, {"x":9, "y":3}, {"x":1.5, "y":4, "w":7}]
      }
```

File: keyboards/kingly_keys/smd_milk/keymaps/default/readme.md [hunk 18]
```diff
@@ -0,0 +1,2 @@
+# Default keymap for 2% Milk
+![Picture](https://i.imgur.com/9PsZ6wa.png)
```
Comment (noroadsleft) [inline keyboards/kingly_keys/smd_milk/keymaps/default/readme.md]: Picture here doesn't match the keymap.

File: keyboards/alpha/config.h [hunk 19]
```diff
@@ -1,5 +1,4 @@
-#ifndef CONFIG_H
-#define CONFIG_H
```
Comment (TheRoyalSweatshirt) [inline keyboards/alpha/config.h]: Whoops, Fixing this

**Comment by TheRoyalSweatshirt** [comment]
Glad I caught something that needed to be fixed on my end.

File: keyboards/kingly_keys/little_foot/little_foot.h [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+    { k31, k32, k33, k34, k35, k36, k37, k38, k39, k310 }, \
+    { XXX, k42, XXX, XXX, XXX, k46, k47, XXX, k49, XXX  }  \
+}
+
+#define LAYOUT_big_space_base( \
+    k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, \
+    k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, \
+    k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, \
+    k31, k32, k33, k34, k35, k36, k37, k38, k39, k310, \
+         k42,              k46,             k49        \
```
Comment (noroadsleft) [inline keyboards/kingly_keys/little_foot/little_foot.h]: `info.json` indicates that this layout uses a 7u spacebar, but you have three keys here. Does the board support a layout without the blockers on the corners?

Comment (TheRoyalSweatshirt) [reply keyboards/kingly_keys/little_foot/little_foot.h]: [Nope!](https://i.imgur.com/ynpS2gg.png) :D

Comment (TheRoyalSweatshirt) [reply keyboards/kingly_keys/little_foot/little_foot.h]: It's 1.5-7-1.5 for the big_space_base

Comment (noroadsleft) [inline keyboards/kingly_keys/little_foot/little_foot.h]: In which case I think `k42` and `k49` need to be deleted on Line 27.

```suggestion
                           k46                         \
```

The way it is now, QMK will expect three keycodes for the bottom row, when there's only one key.

Comment (TheRoyalSweatshirt) [reply keyboards/kingly_keys/little_foot/little_foot.h]: Oh my god I’m an idiot lol

I feel so dumb for not knowing my own layout XD

File: keyboards/kingly_keys/little_foot/keymaps/default/keymap.c [hunk 21]
```diff
@@ -0,0 +1,41 @@
+
+#include QMK_KEYBOARD_H
+#include "quantum.h"
```
Comment (drashna) [inline keyboards/kingly_keys/little_foot/keymaps/default/keymap.c]: This shouldn't be needed, actually, and should be removed:
```suggestion
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,65 @@
...
+      oled_write_P(PSTR("FN\n"), false);
+      break;
+    default:
+      // Or use the write_ln shortcut over adding '\n' to the end of your string
+      oled_write_ln_P(PSTR("Undef"), false);
+  }
+
+  // Host Keyboard LED Status
+  uint8_t led_usb_state = host_keyboard_leds();
+  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: If the OLED is rotated, you may need to shorten this.
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK) ? PSTR("NLCK ") : PSTR("     "), false);
```

Or even:
```suggestion
  oled_write_P(PSTR("NMLCK") , IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK));
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: If the OLED is rotated, you may need to shorten this.
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK) ? PSTR("CAPS ") : PSTR("       "), false);
```

Or even:
```suggestion
  oled_write_P(PSTR("CAPS ") , IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK));
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 24]
```diff
@@ -0,0 +1,65 @@
...
+  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: If the OLED is rotated, you may need to shorten this.
```suggestion
  oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLK") : PSTR("       "), false);
```

Or even:
```suggestion
  oled_write_P(PSTR("SCRLK") , IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK));
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 25]
```diff
@@ -0,0 +1,65 @@
...
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    return OLED_ROTATION_270;  // flips the display 180 degrees if offhand
+}
+
+void oled_task_user(void) {
+  // Host Keyboard Layer Status
+  oled_write_P(PSTR("Let's\nbuild\nsome-\nthing\nto-\nget-\nher!"), false);
+  switch (get_highest_layer(layer_state)) {
+    case BASE:
+      oled_write_P(PSTR("\n"), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]: You can also use `
```suggestion
      oled_write_ln_P(PSTR(""), false);
```

File: keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c [hunk 26]
```diff
@@ -0,0 +1,65 @@
...
+      break;
+    case FN:
+      oled_write_P(PSTR("FN\n"), false);
```
Comment (drashna) [inline keyboards/kingly_keys/romac_plus/keymaps/default/keymap.c]:
```suggestion
      oled_write_ln_P(PSTR("FN"), false);
```

File: keyboards/kingly_keys/little_foot/config.h [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+#define MATRIX_ROW_PINS { F6, B6, B2, B3, B1 }
+#define MATRIX_COL_PINS { F5, F7, B5, B4, E6, D7, C6, D4, D0, D1 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+#define FORCE_NKRO
```
Comment (drashna) [inline keyboards/kingly_keys/little_foot/config.h]: I'm not a fan of this, because it can cause issues, in some cases.

**Pull request merged**